IPython/core/debugger.py

```diff
@@ -621,3 +621,13 @@ def do_where(self, arg):
                 self.print_stack_trace()
 
         do_w = do_where
+
+
+def set_trace(frame=None):
```

Comment:
I could go either way, but I'd like to know what @Carreau thinks when he's online in a couple more hours. :-)